keras_nlp/utils/keras_utils.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def clone_initializer(initializer):
```

Comment:
You can refer to http://google3/third_party/tensorflow_models/official/modeling/tf_utils.py;l=271;rcl=470217720 as an example. The current code might fail if the input is a string.